.fixtures.yml

```diff
@@ -0,0 +1,11 @@
+fixtures:
+  repositories:
+    apache:           'https://github.com/puppetlabs/puppetlabs-apache'
+    concat:           'https://github.com/puppetlabs/puppetlabs-concat'
+    postgresql:       'https://github.com/puppetlabs/puppetlabs-postgresql'
+    redis:            'https://github.com/voxpupuli/puppet-redis'
+    stdlib:           'https://github.com/puppetlabs/puppetlabs-stdlib'
+    systemd:          'https://github.com/camptocamp/puppet-systemd'
+    augeas_core:
```

Comment:
Nit: I like to keep lists sorted so ideally this would be between `apache` and `concat`.